documentation/docs/getting_started/security_101.md

```diff
@@ -2,6 +2,9 @@
 description: This section provides a checklist of steps for running a reliable and secure node.
 image: /img/logo/HornetLogo.png
 keywords:
+description: This section provides a checklist of steps for running a reliable and secure node.
```

Comment:
That looks wrong to me, why is it here twice?

Reply:
```suggestion
```